replication-commands/src/main/java/org/codice/ditto/replication/commands/ConfigDeleteCommand.java

```diff
@@ -115,7 +174,11 @@ void executeWithExistingConfig(final String configName, final ReplicatorConfig c
 
   private boolean removeReplicationItems(String configId) {
```

Comment:
✏️ annotate `configId` with `@Nullable` to clarify it supports `null`

Reply:
Done